concise-mid-tag.cddl

```diff
@@ -139,7 +139,7 @@ tagged-impl-id-type = #6.47115(impl-id-type)
 ; see https://github.com/ietf-rats/ietf-corim-cddl/issues/72
 measurement-map = non-empty<{
   ? comid.ver => version-map
-  ? comid.svn => svn-type
+  ? comid.svn => svn-type-choice
```

Comment:
```suggestion
  ? comid.svn => $svn-type-choice
```

Reply:
OK,  I will make the change and perhaps few other places also to make it look consistent!